docs/pages/make_new_db/getting_started_new_database.rst

```diff
@@ -1,15 +1,35 @@
 Making a New Database
-======================================
+=====================
 
 .. toctree::
-    the_template_schema
-    modifying_schema
+    :titlesonly:
+    :maxdepth: 2
 
+    ../template_schema/template_schema
+    modifying_schema
 
 
 Overview
 --------
 
-1. Make a new Github repository using the `astrodb-template-db <https://github.com/astrodbtoolkit/astrodb-template-db>`_ template repository.
-2. :doc:`Modify the schema <modifying_schema>` to suit your use case.
-3. Ingest data using ``astrodb_utils`` functions.
\ No newline at end of file
+#. Make a new GitHub repository using the `astrodb-template-db <https://github.com/astrodbtoolkit/astrodb-template-db>`_ template repository.
+
+#. Update the ``README.md`` file with your database name and description.
+
+   * Please retain the credit line to the AstroDB Toolkit.
+
+#. Update the ``LICENSE`` file with your license of choice.
```

Comment:
does our license require that they use any specific license?

Reply:
Opened a new issue for this topic.